frog/imports/ui/GraphEditor/SidePanel/fileConvert.jsx

```diff
@@ -0,0 +1,85 @@
+// @flow
```

Comment:
This file is still called `fileConvert`, it makes more sense now to call it back `FileUploader` now :)